eng/Packages.Data.props

```diff
@@ -169,6 +169,11 @@
     <PackageReference Update="Microsoft.Extensions.Options" Version="2.1.0" />
   </ItemGroup>
 
+  <!-- Packages intended for the Moq Extension library only-->
```

Comment:
we should not have 2 entries for Moq.  We should move Moq into its own conditional entry which checks for `$(MSBuildProjectName) == 'Microsoft.Azure.ResourceManager.Moq'` OR `('$(IsTestProject)' == 'true') OR ('$(IsTestSupportProject)' == 'true') OR ('$(IsPerfProject)' == 'true') OR ('$(IsStressProject)' == 'true') OR ('$(IsSamplesProject)' == 'true')`

We should probably create a property for that really long condition and give it a semantic name like `IsTestReleatedProject`

Reply:
I'd prefer that we just mark `Microsoft.Azure.ResourceManager.Moq` as a test project than give it some special logic.   After all, it is a package intended to support testing.

Reply:
Good idea, I will take a look